src/card.tsx

```diff
@@ -121,18 +141,33 @@ type MediaCardProps = {
     button?: RendersNullableElement<typeof ButtonPrimary>;
     buttonLink?: RendersNullableElement<typeof ButtonLink>;
     children?: void;
+    dataAttributes?: DataAttributes;
     'aria-label'?: string;
+    onClose?: () => void;
```

Comment:
![image](https://user-images.githubusercontent.com/6722153/173682860-0d4577e2-7a9e-47ae-a4f8-95452fb28ba0.png)

www.rebrand.ly/577xrw2

close icon goes to Parla when you increase the width of the Media Card

Reply:
se fue a echar gasolina

Reply:
fixed